playwright/e2e/test.niivue.mouse.doubleClick.spec.js

```diff
@@ -0,0 +1,38 @@
+import { test, expect } from '@playwright/test'
```

Comment:
The file names are a bit long-winded (and they were before as well). But at least it is crystal clear what is being tested. Perhaps we can omit the `test.niivue` parts since it is redundant. I do like the other name parts when the pattern is `<subject>.<verb>`

Reply:
@jens-ox - I'm happy to write the tests in TypeScript
@hanayik - I have consolidated the tests under the function or component being used.  Almost all of our end to end tests use niivue with a few exceptions being some of the nvdocument tests. We need a way for the test runner to identify the test files, the [default pattern is .*(test|spec).(js|ts|mjs)](https://playwright.dev/docs/test-configuration). I'm happy to use whatever naming convention.  I think as long as the test names are clear in the test function we can use a shorter naming convention for the file.  After this merge I can rename all of the tests and use the pattern in the future.

Reply:
I think writing tests in TS is ok. Might as well. But make those changes in another PR after this one is merged. I think the naming is ok for now. It's a minor point compared to just making sure the tests do the correct thing :).